### ISSUE TITLE
How to compile

### ISSUE THREAD

**Issue by jsardas** [fixedmachine/projekktor-StrobeMediaPlayback#1]
Hi @fixedmachine ,

Could you please indicate how you are supposed to run the build.sh script? I have already added [flex-install-dir]/bin to the PATH system variable at the beginning of the script, but I'm still unable to compile.

Thanks.

**Comment by rwlodkowski** [comment]
@jsardas what OS are you using?

**Comment by jsardas** [comment]
I'm using Linux with the Apache Flex SDK...

**Comment by rwlodkowski** [comment]
Could you send me console output after executing `./build.sh` ?

**Comment by jsardas** [comment]
I believe my issue is related to the Flex SDK version I'm using, but I'm not sure...

The console output shows errors when the script tries to compile for v 11.4, such as:

`/root/apache-flex-sdk-4.13.0-bin/frameworks/flex-config.xml(65): Error: unable to open 'libs/player/11.4/playerglobal.swc'`

`/root/apache-flex-sdk-4.13.0-bin/frameworks/flex-config.xml(85): Error: unable to open 'libs/player/11.4'`

But at the end it succeeds when compilation is for v 10.2, such as:

`Compiling StrobeMediaPlayback_debug.swf - projekktor-StrobeMediaPlayback debug version target 10.2 for Linux:
Loading configuration file /root/apache-flex-sdk-4.13.0-bin/frameworks/flex-config.xml
/root/projekktor-StrobeMediaPlayback/bin/debug/StrobeMediaPlayback_target_10.2.swf (544952 bytes)`

I noticed this is defined in your build.sh script as follows:

**For latest release of Adobe Flash for Linux**
`TARGET_10_2_OPT="-target-player=10.2 -swf-version=11"`

**For newest Adobe Flash**
`TARGET_11_4_OPT="-target-player=11.4 -swf-version=17"`

Could you help me determine what I'm missing?

Thanks.

**Comment by jsardas** [comment]
Upon further testing, my error was indeed related to the Flex SDK version and the different target player versions I was using. I was able to solve this and compiled the project successfully.

Something I still can't solve is how to compile with HLS/DASH based on the conditional reference at:
`https://github.com/fixedmachine/projekktor-StrobeMediaPlayback/commit/1125056809cef542a28af4ea2d9784ab4b4aaf6f`

I made sure the flashls and dashas submodules were properly cloned in addition to your project, and then added the following to build.sh at the end of COMMON_OPT:

`-define=CONFIG::HLS,true`

However it doesn't seem to be working based on the tests I did with the compiled swf trying to load an HLS feed. I can confirm it's not a configuration issue since I tested with a pre-compiled "StrobeMediaPlayback_hls_mss.swf", and the HLS feed played without issues.

Am I doing something wrong at build.sh for adding HLS support? Is a different syntax required?

Thanks a lot for your time.
